Branch v0.7-devel (06e74656f66d421c14106f5dbe6789d92435fd1c) fails to build.

git clone https://github.com/jthornber/thin-provisioning-tools.git
    cd thin-provisioning-tools/
    git checkout origin/v0.7-devel
    autoconf && ./configure && make
    ...
    [LD]  bin/pdata_tools
    thin-provisioning/shared_library_emitter.o: In function `thin_provisioning::create_custom_emitter(std::string const&, std::ostream&)':
    /home/thanos/thin-provisioning-tools/thin-provisioning/shared_library_emitter.cc:15: undefined reference to `dlopen'
    /home/thanos/thin-provisioning-tools/thin-provisioning/shared_library_emitter.cc:19: undefined reference to `dlerror'
    /home/thanos/thin-provisioning-tools/thin-provisioning/shared_library_emitter.cc:20: undefined reference to `dlsym'
    /home/thanos/thin-provisioning-tools/thin-provisioning/shared_library_emitter.cc:22: undefined reference to `dlerror'
    /home/thanos/thin-provisioning-tools/thin-provisioning/shared_library_emitter.cc:17: undefined  reference to `dlerror'
    collect2: error: ld returned 1 exit status
    make: *** [bin/pdata_tools] Error 1

--- a/Makefile.in
+++ b/Makefile.in
@@ -137,7 +137,7 @@ CXXFLAGS+=@CXXDEBUG_FLAG@
 CXXFLAGS+=@CXX_STRERROR_FLAG@
 CXXFLAGS+=@LFS_FLAGS@
 INCLUDES+=-I$(TOP_BUILDDIR) -I$(TOP_DIR) -I$(TOP_DIR)/thin-provisioning
-LIBS:=-laio -lexpat
+LIBS:=-laio -lexpat -ldl
 
 ifeq ("@DEVTOOLS@", "yes")
 LIBS+=-lncurses

